src/Sentry/SentryClient.cs

```diff
@@ -94,6 +94,44 @@ public SentryId CaptureEvent(SentryEvent? @event, Scope? scope = null)
             }
         }
 
+        /// <summary>
+        /// Captures a manually created user feedback and sends it to Sentry.
+        /// </summary>
+        /// <param name="userFeedback">The user feedback to send to Sentry.</param>
+        public void CaptureUserFeedback(SentryUserFeedback userFeedback)
+        {
+            if (_disposed)
+            {
+                throw new ObjectDisposedException(nameof(SentryClient));
+            }
+
+            if (userFeedback.EventId.Equals(SentryId.Empty))
+            {
+                //Ignore the userfeedback if EventId is empty
+                _options.DiagnosticLogger?.LogInfo(
+                        "User feedback discarted due to because no event is associated.", userFeedback.GetType());
+                return;
+            }
+            else if ( string.IsNullOrWhiteSpace(userFeedback.Email) ||
+                 string.IsNullOrWhiteSpace(userFeedback.Comments))
+            {
+                //Ignore the userfeedback if a required field is null or empty.
+                _options.DiagnosticLogger?.LogInfo(
+                        "User feedback discarted due to one or more properties being empty.", userFeedback.GetType());
+                return;
+            }
+            var envelope = Envelope.FromUserFeedback(userFeedback);
+
+            if (Worker.EnqueueEnvelope(envelope))
```

Comment:
Dont' we have a private CaptureEnvelope here or something?

I assume this if and the block after is a copy paste from another method right? Rather not letting them go out of sync

Reply:
It was based on DoSendEvent https://github.com/getsentry/sentry-dotnet/blob/main/src/Sentry/SentryClient.cs#L168

From what I saw, there are no CaptureEnvelope callbacks, and the above code seems to be the nearest code related to capturing Envelopes.

Reply:
But you're in the `SentryClient` so you could refactor this to a private method and call from both places instead of cooying these lines